### ISSUE TITLE
Makefile error - pgxs.mk missing

### ISSUE THREAD

**Issue by MarzoLinus** [tds-fdw/tds_fdw#228]
Hello,
I have a Postgres installation on my Debian 10 (buster) VPS, with Postgres 11 already up & running.
I need to connect to a SQL Server DB so I:
1) installed libsybdb5 freetds-dev freetds-common
2) downloaded tds_fdw-1.0.8
3) extracted the archive
4) launched 'make USE_PGXS=1'

I find the following error:
`Makefile:53: /usr/lib/postgresql/12/lib/pgxs/src/makefiles/pgxs.mk: No such file or directory
make: *** No rule to make target '/usr/lib/postgresql/12/lib/pgxs/src/makefiles/pgxs.mk'.  Stop.
`
The problem could seem that I have Postgres 11 installed, not 12.
But even looking in the same path changing '12' with '11' I am not able to find that file, so I can't build the fdw..

It looks like a trivial question, but I cannot find a solution.

Any help really appreciated.

**Comment by MarzoLinus** [comment]
I resolved installing postgresql-server-dev-11.

Now I get another error, will open a specific issue.

This can be closed, but I think that postgresql-server-dev-xx or postgresql-server-dev-all should be indicated as pre-requirements.

**Comment by juliogonzalez** [comment]
It's on the instructions for Ubuntu (basically the same as for Debian): https://github.com/tds-fdw/tds_fdw/blob/master/InstallUbuntu.md#install-postgresql

> If you need to install PostgreSQL, do so by following the apt installation directions. For example, to install PostgreSQL 9.5 on Ubuntu:
> [...]
> sudo apt-get install postgresql-9.5 postgresql-client-9.5 postgresql-server-dev-9.5

However maybe you can contribute a new markdown file specific for Debian, as the Ubuntu instructions show how to add the repository for Ubuntu.

**Comment by MarzoLinus** [comment]
Sorry, I skipped that part without paying attention because I already had Postgres installed.

I will do a .md file for Debian10, as soon as I succeed in the installation (now I am blocked by another issue, trying to investigate about it).

Thank you!
